qualtran/surface_code/ccz2t_cost_model.py

```diff
@@ -189,7 +189,9 @@ def get_ccz2t_costs(
     )
     failure_prob = distillation_error + data_error
     footprint = factory.footprint() + data_block.footprint(n_algo_qubits=n_algo_qubits)
-    duration_hr = (cycle_time_us * n_cycles) / (1_000_000 * 60 * 60)
+    duration_hr = (cycle_time_us * n_cycles * data_block.n_timesteps_to_consume_a_magic_state()) / (
```

Comment:
is this correct? if it takes e.g. 7 steps (7*d cycles) to make one magic state and 7 steps to consume a magic state the total time is still just n_cycles * cycle_time

Reply:
hah, we define `n_cycle` to be the total number of surface code cycles so it's `time_steps * $d$ * n_timesteps_to_consume_magic_state`. I will update the PR accordingly.

Reply:
actually we need to take the `max`. I renamed `n_cycle` to `n_generation_cycles` and redefined `n_cycle` as `max(n_generation_cycles, n_consumption_cycles)`